### DISCUSSION TITLE
Error building DATASET and print heatmaps annotations in PoseC3D

### DISCUSSION THREAD

**Discussion by christina05011** [open-mmlab/mmaction2#2559] [Q&A]
Hi, I have a problem when I am building datasets in PoseC3D. 

First, I did the steps according to the [tutorial ](https://colab.research.google.com/github/open-mmlab/mmaction2/blob/master/demo/mmaction2_tutorial.ipynb#scrollTo=dDBWkdDRk6oz), but I want to print only heatmaps annotations, for that reason I run:

```
dataloader_cfg = copy.deepcopy(cfg.train_dataloader)
dataset_cfg = dataloader_cfg.pop('dataset')
dataset = DATASETS.build(dataset_cfg)
print(next(iter(dataset)))
```

When I run those commands with config tsn_imagenet-pretrained-r50_8xb32-1x1x3-100e_kinetics400-rgb.py, I have results resized and not empty. 

![image](https://github.com/open-mmlab/mmaction2/assets/42581109/0d314823-91be-495c-a59f-8193334d2b63)

But when I run with config slowonly_r50_8xb16-u48-240e_ntu60-xsub-keypoint.py, I only have zeros. Also, when I train, I only have zeros too. 

![image](https://github.com/open-mmlab/mmaction2/assets/42581109/0da65fac-5ab2-499e-917f-defefa29362b)

- How can I print only heatmaps annotations? Did I do it correctly?
- Why does it only print zeros (none data)?

Thank you for the answers.

**Comment by cir7** [comment]
same topic as #2560
